webapp/app/js/perf.js

```diff
@@ -222,6 +237,20 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
         }
         $scope.$digest();
       });
+
+      //Highlight the points persisted in the url
+      var showResetRevButton = false;
+      $scope.seriesList.forEach(function(series, i) {
+        if (series.highlighted.length > 0 && series.visibility) {
+            showResetRevButton = true;
+            $scope.plot.highlight(i, series.highlighted[0]);
+            $('#findRevText').val(series.highlighted[1]);   
+        }
+      });
+
+      if (showResetRevButton) {
+        $(".resetHighlightButton").show();
```

Comment:
It would be more angular-like to have a variable like $scope.showResetRevButton, then make displaying the button conditional on it being true using the ngShow directive:  https://docs.angularjs.org/api/ng/directive/ngShow